sql/api/pom.xml

```diff
@@ -40,6 +40,11 @@
             <artifactId>spark-common-utils_${scala.binary.version}</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.spark</groupId>
+            <artifactId>spark-unsafe_${scala.binary.version}</artifactId>
```

Comment:
Which one requires this?

Reply:
`def outOfDecimalTypeRangeError(str: UTF8String): SparkArithmeticException`

Reply:
`UTF8String` stays into `spark-unsafe`.